pkgs/development/libraries/numcpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "numcpp";
+  version = "2.11.0";
+
+  src = fetchFromGitHub {
+    owner = "dpilger26";
+    repo = "NumCpp";
+    rev = "Version_${finalAttrs.version}";
+    hash = "sha256-IAku1bcaMkawZxpQbvxcS6VX07ogw4UGo1DX2Wa8xwU=";
+  };
+
+  nativeBuildInputs = [cmake];
+
+  buildInputs = [boost];
+
+  postInstall = ''
+    sed -i s/\$\{PACKAGE_PREFIX_DIR}// $out/share/NumCpp/cmake/NumCppConfig.cmake
```

Comment:
For simple substitutions we tend to prefer `substituteInPlace` but it's not a major issue.

Reply:
okay i moved to that method